docs/en/connector-v2/sink/AmazonSqs.md

```diff
@@ -0,0 +1,45 @@
+# AmazonSqs
+
+> Amazon SQS sink connector
+
+## Support Those Engines
```

Comment:
Add titles for supported versions : `localstack/localstack:0.11.2`

Reply:
localstack is not a engine but only used for testing purpose.